src/components/DpButtonIcon/DpButtonIcon.vue

```diff
@@ -37,7 +37,7 @@ export default {
 
   computed: {
     translatedText () {
-      return Translator.trans(this.text)
+      return this.text
```

Comment:
The computed only exists because of the Translator. You can as well use the prop in the template directly when dropping Translator.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/526/commits/3dc2e1494d09cc79169c32d0d4582d3793567d8a